packages/next/src/server/app-render/use-flight-response.tsx

```diff
@@ -57,28 +57,6 @@ export function useFlightStream<T>(
   return newResponse
 }
 
-/**
- * There are times when an SSR render may be finished but the RSC render
- * is ongoing and we need to wait for it to complete to make some determination
- * about how to handle the render. This function will drain the RSC reader and
- * resolve when completed. This will generally require teeing the RSC stream and it
- * should be noted that it will cause all the RSC chunks to queue in the underlying
- * ReadableStream however given Flight currently is a push stream that doesn't respond
- * to backpressure this shouldn't change how much memory is maximally consumed
- */
-export async function flightRenderComplete(
-  flightStream: ReadableStream<Uint8Array>
-): Promise<void> {
-  const flightReader = flightStream.getReader()
-
-  while (true) {
-    const { done } = await flightReader.read()
-    if (done) {
-      return
-    }
-  }
-}
```

Comment:
no longer needed because we wait for the flight render to complete before we check dynamic so we can capture the flight render as part of metadata